ui/src/app/components/terms-of-service.tsx

```diff
@@ -132,6 +132,7 @@ export class TermsOfService extends React.Component<TosProps, TosState> {
             </div>
             <div>
               <CheckBox
+                aria-label='Acknowledge Terms'
                 data-test-id='agreement-check'
```

Comment:
do we still need data-test-id if its being replaced in test with input aria-label

Reply:
We should not, but I didn't want to risk taking them out just yet. I'll see about doing another PR to take out any of these that I can to help establish the new pattern.